src/ledgerwriter/src/main/java/anthos/samples/bankofanthos/ledgerwriter/LedgerWriterController.java

```diff
@@ -72,6 +73,9 @@
     public static final String UNAUTHORIZED_CODE = "not authorized";
     public static final String JWT_ACCOUNT_KEY = "acct";
 
+    @Autowired
```

Comment:
To be clear, is this change required to enable tracing?

Reply:
Yup it's to enable tracing, the restTemplate needs to be autowired to allow Spring Cloud sleuth to inject an Interceptor to the request.

Reply:
Also the fix regarding paging was because in order to enable tracing I had to update the spring boot version and the PageRequest was deprecated which is why I had to update the function call.

Reply:
Thanks for the explanation.  Everything sounds good.